platform/broadcom/sai.mk

```diff
@@ -1,8 +1,8 @@
-BRCM_SAI = libsaibcm_4.2.1.5-4_amd64.deb
-$(BRCM_SAI)_URL = "https://sonicstorage.blob.core.windows.net/packages/bcmsai/4.2/master/libsaibcm_4.2.1.5-4_amd64.deb?sv=2015-04-05&sr=b&sig=dHfQf95FxgLBlnrGJwv6pIkAYAc6mOGbz9vvUMfxFvg%3D&se=2034-07-28T05%3A53%3A21Z&sp=r"
-BRCM_SAI_DEV = libsaibcm-dev_4.2.1.5-4_amd64.deb
+BRCM_SAI = libsaibcm_4.2.1.5_amd64-buster.deb
+$(BRCM_SAI)_URL = "https://sonicstorage.blob.core.windows.net/packages/bcmsai/4.2/master/libsaibcm_4.2.1.5_amd64-buster.deb?sv=2019-12-12&st=2020-11-24T21%3A45%3A37Z&se=2021-11-25T21%3A45%3A00Z&sr=b&sp=r&sig=6we05VowbABC4VhxUOQlZdnfohRgqpOMUxYdL9O4sX0%3D"
+BRCM_SAI_DEV = libsaibcm-dev_4.2.1.5_amd64-buster.deb
```

Comment:
can you put buster in the url path/folder?

Reply:
Sure. Will update.

Reply:
Done